pom.xml

```diff
@@ -6,13 +6,13 @@
 
     <groupId>org.bstats</groupId>
     <artifactId>bstats</artifactId>
+    <version>1.4-SNAPSHOT</version>
     <packaging>pom</packaging>
-    <version>1.3</version>
 
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    </properties>
+    <name>BStats-Metrics</name>
```

Comment:
The project's name is written `bStats`, not `BStats`

Reply:
right